config/prepare.js

```diff
@@ -1,3 +1,39 @@
+const redColor = '\x1b[31m';
```

Comment:
How about using [`chalk`](https://github.com/chalk/chalk)? (it's already a dependency of a bunch of tools, so it might not even add disk space) Added benefit: it automatically detects if colors are supported.

nit: I've seen `\033` used much more times than `\x1b`. YMMV.

Reply:
done, chalk used